src/main/Tournament.java

```diff
@@ -0,0 +1,46 @@
+package main;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Component
+@Data
+@RequiredArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Tournament {
+    List<Team> teamList;
+    List<Race> raceList = new ArrayList<>();
```

Comment:
This class field can live in method.
No need to perform operations from `Controller`.